### PR TITLE
Don't show no patients found error view when search query is empty

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.shortcut.com/simpledotorg/story/8601/don-t-show-no-patients-found-error-view-when-search-query-is-empty

### CODE REVIEW

**Comment by msasikanth** [comment]
Current dependencies on/for this PR:
* master
  * **PR #4021** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/4021" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/4021?utm_source=stack-comment).

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,8 @@
 - Bump coroutines to v1.6.3
 - Implement view effects to show overdue search results
 - Wire Mobius loop with overdue search screen UI
+- Don't show no patients found error view when search query is empty
+- Prefill search query when search history item is clicked
```
Comment (janhavisinghh) [inline CHANGELOG.md:26]: We need to remove the extra change log here

Comment (msasikanth) [reply CHANGELOG.md:26]: I think the dependency PR is merged in first

**Pull request closed without merging**